src/Nancy/Extensions/ModuleExtensions.cs

```diff
@@ -12,31 +9,22 @@ namespace Nancy.Extensions
     public static class ModuleExtensions
     {
         /// <summary>
-        /// A regular expression used to manipulate parameterized route segments.
-        /// </summary>
-        /// <value>A <see cref="Regex"/> object.</value>
-        [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-        private static readonly Regex ModuleNameExpression =
-            new Regex(@"(?<name>[\w]+)Module$", RegexOptions.Compiled);
-
-        /// <summary>
         /// Extracts the friendly name of a Nancy module given its type.
         /// </summary>
         /// <param name="module">The module instance</param>
         /// <returns>A string containing the name of the parameter.</returns>
-        /// <exception cref="FormatException"></exception>
         public static string GetModuleName(this INancyModule module)
         {
             var typeName = module.GetType().Name;
-            var nameMatch =
-                ModuleNameExpression.Match(typeName);
 
-            if (nameMatch.Success)
+            var offset = typeName.LastIndexOf("Module", StringComparison.Ordinal);

```

Comment:
Would be a breaking change since the regex was not case sensitive. Decided to leave it as `Ordinal` as the goal of the change was performance improvements, not behavior change